website/docs/language/state/examples/encryption/external-method/method-external-method.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+import base64
+import json

```

Comment:
Perhaps the python and go examples should show both encryption and decryption?  I know it's trivial to assume how it's supposed to work from the docs, but it could be useful.

Reply:
Your examples earlier up showed using the same command, but with different flags for encrypt vs decrypt.  I was trying to point out that it could be confusing to not have the difference shown in the example programs.